specification/securityinsights/resource-manager/readme.python.md

```diff
@@ -15,13 +15,27 @@ python:
   package-version: 0.1.0
   clear-output-folder: true
 ```
+``` yaml $(python) && $(track2)
```

Comment:
Why do we need all of those line? its looks like a duplication of the first block

Reply:
it's needed for releasing track2 version of python